The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/11018398902/job/30598588959) [(plain)](https://github.com/rust-lang/rust/commit/4ea54fc7d63153d4a8abbed7a5dec1d6dec6c6ab/checks/30598588959/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
#16 2.914 Building wheels for collected packages: reuse
#16 2.915   Building wheel for reuse (pyproject.toml): started
#16 3.175   Building wheel for reuse (pyproject.toml): finished with status 'done'
#16 3.177   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#16 3.177   Stored in directory: /tmp/pip-ephem-wheel-cache-ekn2lpy7/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#16 3.180 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#16 3.594 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#16 3.595 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#16 DONE 3.7s
---
    Checking rustc-std-workspace-core v1.99.0 (/checkout/library/rustc-std-workspace-core)
error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
    | |        ------------------------------------ in this macro invocation (#2)
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
    | |        ------------------------------------ in this macro invocation (#2)
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
5   | | $macro!( Relaxed, 8, __aarch64_cas8_relax );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
5   | | $macro!( Relaxed, 8, __aarch64_cas8_relax );
6   | | $macro!( Acquire, 1, __aarch64_cas1_acq );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
7   | | $macro!( Acquire, 2, __aarch64_cas2_acq );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
8   | | $macro!( Acquire, 4, __aarch64_cas4_acq );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
9   | | $macro!( Acquire, 8, __aarch64_cas8_acq );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
10  | | $macro!( Release, 1, __aarch64_cas1_rel );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
11  | | $macro!( Release, 2, __aarch64_cas2_rel );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
2   | | $macro!( Relaxed, 1, __aarch64_cas1_relax );
3   | | $macro!( Relaxed, 2, __aarch64_cas2_relax );
4   | | $macro!( Relaxed, 4, __aarch64_cas4_relax );
...   |
12  | | $macro!( Release, 4, __aarch64_cas4_rel );
...   |
...   |
17  | | $macro!( AcqRel, 8, __aarch64_cas8_acq_rel );
18  | | }; }
    | |____- in this expansion of `foreach_cas!` (#1)
note: found an item that was configured out
   --> /checkout/library/core/src/arch.rs:30:11
    |
    |
30  | pub macro naked_asm("assembly template", $(operands,)* $(options($(option),*))?) {
note: the item is gated here
   --> /checkout/library/core/src/arch.rs:29:1
    |
    |
29  | #[cfg(not(bootstrap))]

error[E0433]: failed to resolve: could not find `naked_asm` in `arch`
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.127/src/aarch64_linux.rs:139:38
    |
    |
130 | / macro_rules! compare_and_swap {
131 | |     ($ordering:ident, $bytes:tt, $name:ident) => {
132 | |         intrinsics! {
133 | |             #[maybe_use_optimized_c_shim]
...   |
139 | |                 unsafe { core::arch::naked_asm! {
    | |                                      ^^^^^^^^^ could not find `naked_asm` in `arch`
156 | |     };
157 | | }
    | |_- in this expansion of `compare_and_swap!` (#2)
...
...
267 |   foreach_cas!(compare_and_swap);
    |
   ::: /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-std/aarch64-unknown-linux-gnu/release/build/compiler_builtins-69c6c11ac4925ba4/out/outlined_atomics.rs:1:17
    |
    |
1   |   #[macro_export] macro_rules! foreach_cas { ($macro:path) => {
    |  _________________-
```

</details>

--- a/library/alloc/Cargo.toml
+++ b/library/alloc/Cargo.toml
@@ -10,7 +10,7 @@ edition = "2021"
 
 [dependencies]
 core = { path = "../core" }
-compiler_builtins = { version = "0.1.126", features = ['rustc-dep-of-std'] }
+compiler_builtins = { version = "0.1.130", features = ['rustc-dep-of-std'] }
 
 [dev-dependencies]
 rand = { version = "0.8.5", default-features = false, features = ["alloc"] }

--- a/library/std/Cargo.toml
+++ b/library/std/Cargo.toml
@@ -17,7 +17,7 @@ cfg-if = { version = "1.0", features = ['rustc-dep-of-std'] }
 panic_unwind = { path = "../panic_unwind", optional = true }
 panic_abort = { path = "../panic_abort" }
 core = { path = "../core", public = true }
-compiler_builtins = { version = "0.1.126" }
+compiler_builtins = { version = "0.1.130" }
 profiler_builtins = { path = "../profiler_builtins", optional = true }
 unwind = { path = "../unwind" }
 hashbrown = { version = "0.14", default-features = false, features = [

